task_runner.rb

```diff
@@ -24,13 +24,15 @@ def run_tasks
       tasks.sort!
     end
 
+    project = KanboardProject.new('id' => project_id)
     tasks.each do |task|
-      project = KanboardProject.new('id' => project_id)
       task_configuration = profile.task_configuration.fetch(task_name(task), {})
 
       logger.info "Starting task #{task}"
+      start = Time.now
```

Comment:
Same here

Reply:
This is intentional to tell how long a task takes. The other is useful for debugging the requests, but generates two lines for every request so I left it commented out.